config.yml

```diff
@@ -7723,3 +7441,355 @@ production_recipes:
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 13
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 13
+  Iron_XP_Recipe_3:
+    name: Craft XP Bottles 3
+    production_time: 10
+    inputs:
+      Cactus:
+        material: CACTUS
+        amount: 256
+      Cocoa:
+        material: INK_SACK
+        durability: 3
+        amount: 256
+      Baked Potato:
+        material: BAKED_POTATO
+        amount: 320
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 16
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 16
+  Diamond_Crate:
+    name: Package Diamond Crate
+    production_time: 20
+    inputs:
+      Iron Crate:
+        material: POWERED_MINECART
+        amount: 2
+        display_name: Iron Crate
+        lore: Processed market supplies. It can be converted into XP but it's not very efficient...
+      Golden Carrot:
+        material: GOLDEN_CARROT
+        amount: 64
+      Mycelium:
+        material: MYCEL
+        amount: 128
+      Pumpkin Pie:
+        material: PUMPKIN_PIE
+        amount: 64
+      Compact Ice:
+        material: ICE
+        amount: 256
+      Acacia Wood:
+        material: LOG_2
+        durability: 0
+        amount: 320
+      Melon:
+        material: MELON_BLOCK
+        amount: 256
+      Shrub:
+        material: LONG_GRASS
+        durability: 1
+        amount: 128
+    outputs:
+      Diamond Crate:
+        material: ENCHANTMENT_TABLE
+        amount: 1
+        display_name: Diamond Crate
+        lore: A large assortment of global resources. Nothing fancy. It certainly has a lot of XP value.
+  Convert_Iron_Crate:
+    name: Convert Iron Crate into XP
+    production_time: 30
+    fuel_time: 30
+    inputs:
+      Iron Crate:
+        material: POWERED_MINECART
+        amount: 1
+        display_name: Iron Crate
+        lore: Processed market supplies. It can be converted into XP but it's not very efficient...
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 64
+    outputs:
+      Emerald:
+        materials: EMERALD
+        amount: 23
+  Emerald_Crate:
+    name: Package Emerald Crate
+    production_time: 15
+    inputs:
+      Diamond Crate:
+        material: ENCHANTMENT_TABLE
+        amount: 2
+        display_name: Diamond Crate
+        lore: A large assortment of global resources. Nothing fancy. It certainly has a lot of XP value.
+      Ink Sac:
+        material: INK_SACK
+        durability: 0
+        amount: 16
+      Poisonous Potato:
+        material: POISONOUS_POTATO
+        amount: 32
+      Podzol:
+        material: DIRT
+        durability: 2
+        amount: 128
+      Rose:
+        material: RED_ROSE
+        durability: 0
+        amount: 192
+      Bucket of Lava:
+        material: LAVA_BUCKET
+        amount: 16
+      Obsidian:
+        material: OBSIDIAN
+        amount: 384
+      Eye of Ender:
+        material: EYE_OF_ENDER

```

Comment:
Yup, I found that out yesterday after reviewing the recipe for eye of ender and realized that this works out pretty well. Yes, you are correct, the Aspect Elementizer has no real use use after it is constructed (a shame, really. No factory should be useless after a single purpose) aside for making the Eyes of Ender. I imagine with that ingredient in the Emerald Cauldron recipe, it'll seriously bring to question what groups of players can actually afford to go all the way to the top, and what groups have to take charge in cooperating to get their input quotas reached.